.github/workflows/build_test.yml

```diff
@@ -0,0 +1,41 @@
+name: ROS2 CI
+
+on:
+  pull_request:
+    branches:
+      - 'develop'
+      - 'main'
+
+jobs:
+  test_environment:
+    runs-on: [ubuntu-latest]
+    strategy:
+      fail-fast: false
+      matrix:
+        ros_distribution:
+          - foxy
+          - rolling
+        include:
+          # Foxy Fitzroy (June 2020 - May 2023)
+          - docker_image: rostooling/setup-ros-docker:ubuntu-focal-ros-foxy-base-latest
+            ros_distribution: foxy
```

Comment:
`ros-foxy-ros-base`

Reply:
i found some from `osrf` too...would you not want to use those ones?

Reply:
https://hub.docker.com/r/osrf/ros/tags?page=1&ordering=last_updated

Reply:
nevermind looks like it doesnt work.